mod/ckeditor/views.php

```diff
@@ -2,7 +2,6 @@
 
 return [
 	'default' => [
-		'ckeditor.js' => __DIR__ . '/vendors/ckeditor/ckeditor.js',
```

Comment:
You did it in your PR, too. That's where I got the idea from.

To what extend is BC? I keep elgg/ckeditor in place (only changing the requirement within it). So, anyone requiring elgg/ckeditor directly anywhere else shouldn't have a BC issue.